api/v1beta1/kogitoruntime_types.go

```diff
@@ -119,7 +119,8 @@ type KogitoRuntimeList struct {
 func (k *KogitoRuntimeList) GetItems() []api.KogitoRuntimeInterface {
 	models := make([]api.KogitoRuntimeInterface, len(k.Items))
 	for i, v := range k.Items {
-		models[i] = &v
+		item := v
```

Comment:
does it change something?

Reply:
when we directly assign `models[i] = &v` then pointer value of v is assign to `models[i]` so in the next iteration when value of v is change then value inside `models[i]` also override.
With `item` we had introduced local variable which create new for every iteration so value inside `models[i]` never change.

Reply:
good catch, thanks.